pkg/metacni/metacni.go

```diff
@@ -269,6 +304,10 @@ func createDelegatedInterface(syncher *syncher.Syncher, danmClient danmclientset
   if err != nil {
     syncher.PushResult(iface.Network, errors.New("DanmEp object could not be PUT to K8s due to error:" + err.Error()), nil)
   }
+  err = danmep.SetDanmEpSysctls(ep)
+  if err != nil {
+    syncher.PushResult(iface.Network, errors.New("Sysctls could not be set due to error:" + err.Error()), nil)
```

Comment:
it might make me silly, but this code made me thinking... shouldn't we immediately return after every failed syncher.PushResult calls? :)

Reply:
Done

Reply:
haven't you forgot to git push?